jaxrs-clients/src/main/java/com/palantir/remoting2/jaxrs/feignimpl/FeignSerializableErrorErrorDecoder.java

```diff
@@ -42,7 +54,68 @@ public Exception decode(String methodKey, Response response) {
         } catch (IOException e) {
             return new RuntimeException("Cannot get input stream from response: " + e.getMessage(), e);
         }
-        return SerializableErrorToExceptionConverter.getException(contentTypes, response.status(), response.reason(),
+        Exception exception = SerializableErrorToExceptionConverter.getException(
```

Comment:
Not exactly sure what you mean by this. We use this exception in the `RetryableException`